3rdparty/carotene/hal/tegra_hal.hpp

```diff
@@ -1846,7 +1862,7 @@ TegraCvtColor_Invoker(bgrx2hsvf, bgrx2hsv, src_data + static_cast<size_t>(range.
 #undef cv_hal_cvtBGRtoHSV
 #define cv_hal_cvtBGRtoHSV TEGRA_CVTBGRTOHSV
 #undef cv_hal_cvtTwoPlaneYUVtoBGR
-#define cv_hal_cvtTwoPlaneYUVtoBGR TEGRA_CVT2PYUVTOBGR
+#define cv_hal_cvtTwoPlaneYUVtoBGR TEGRA_CVT2PYUVTOBGR_OVERLOADED
```

Comment:
HAL interface functions should not have overloads.
Because used processor macros don't support overloading.
So new name should be added for implementation with new parameters/behavior.

(Check is following: code should compile and work without "carotene" changes)

/cc @mshabunin @vpisarev

Reply:
Should now be fixed. Have added the new function under another name. Probably another option was to use variadic macro, as the number of parameters differs between the two functions, but assessment of feasibility of this would require understanding of initial reasoning.